lib/ui/window/viewport_metrics.cc

```diff
@@ -38,12 +37,7 @@ ViewportMetrics::ViewportMetrics(double p_device_pixel_ratio,
           p_physical_system_gesture_inset_right),
       physical_system_gesture_inset_bottom(
           p_physical_system_gesture_inset_bottom),
-      physical_system_gesture_inset_left(p_physical_system_gesture_inset_left) {
-  // Ensure we don't have nonsensical dimensions.
-  FML_DCHECK(physical_width > 0);
```

Comment:
Why are we removing these checks?

Reply:
I added these in the original Opacity PR due to feedback from amott.

However, they were causing failures when running the Android benchmark for me.  I grew concerned about the impact on other platforms (iOS, windows) and removed them

Reply:
Using `vector` sounds good. I'm just curious because we were including `<stack>` but never using it  :)

Reply:
Oops, the above comment is in the wrong thread... Does `FML_DCHECK(physical_width >= 0)` solve the failures? If so, maybe we can still use them to catch negative dimensions. Also, as you mentioned in another thread, we can remove the changes in FlutterView.java due to the changes here.

Reply:
Yes, >= solves it so I will do that